app/server/index.js

```diff
@@ -58,19 +58,27 @@ app.use(compression());
 app.use(bodyParser.urlencoded());
 app.use(bodyParser.json());
 
-app.use("/css/leaflet.css", express.static(path.join(__dirname, '../../node_modules/leaflet/dist/leaflet.css')));
+app.use("/leaflet.css", express.static(path.join(__dirname, '../../node_modules/leaflet/dist/leaflet.css')));
 app.use("/socket.io.js", express.static(path.join(__dirname, '../../node_modules/socket.io/node_modules/socket.io-client/socket.io.js')));
-app.use("/css", express.static(path.join(__dirname, '../client/css')));
-app.use("/images", express.static(path.join(__dirname, '../client/images')));
+app.use("/Map", express.static(path.join(__dirname, '../clients/Map')));
+app.use("/Admin", express.static(path.join(__dirname, '../clients/Admin')));
+app.use("/_common", express.static(path.join(__dirname, '../clients/_common')));
 
 app.get('/', function(req, res){
-    res.sendFile(path.join(__dirname, '../client/index.html'));
+    res.sendFile(path.join(__dirname, '../clients/Map/index.html'));
 });
 
-app.get('/browserify-bundle.js', function(req, res){
-    res.sendFile(path.join(__dirname, '../client/browserify-bundle.js'));
+app.get('/Admin', function(req, res){
```

Comment:
I tend to agree with this, but it would mean that twilio (or equivalent) should be forwarding the sms to two different domains, wouldn't it ?
Indeed, Admin will need to have access to the sms sent, as much as the regular Map does, so that we know the ant status.
Is this possible/recommended ?